src/Controller/CRUDController.php

```diff
@@ -183,7 +179,7 @@ public function batchActionDelete(ProxyQueryInterface $query)
      *
      * @return Response|RedirectResponse
      */
-    public function deleteAction(Request $request)
+    public function deleteAction(Request $request): Response
```

Comment:
On few methods which return `Response|RedirectResponse` I put `Response` as a return type since `RedirectResponse` extends `Response` but not sure if there are reasons against that

Reply:
I would say that you can remove the @return too then.

`Response|RedirectResponse` is not needed.